docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -0,0 +1,129 @@
+# Incremental sync
```

Comment:
do we want to document the use of `stream_interval` anywhere?

Reply:
Good call, added an info box for that and updated the tutorial